__tests__/components/banner/PrivacyPolicy.test.tsx

```diff
@@ -0,0 +1,37 @@
+import "@testing-library/jest-dom";
+import PrivacyPolicyBanner from "@/components/banner/PrivacyPolicy";
+import { renderWithRedux } from "@__mocks__/renderWithRedux";
+import { fireEvent, screen } from "@testing-library/react";
+
+describe("PrivacyPolicyBanner", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should render PrivacyPolicy", () => {
+    renderWithRedux(<PrivacyPolicyBanner />, {
+      banner: { showCookiePolicy: true },
+    });
+    const privacyPolicy = screen.getByTestId("PrivacyPolicy");
+    expect(privacyPolicy).toBeInTheDocument();
+  });
+
+it("should render the close button", () => {
+  const { getByTestId } = renderWithRedux(<PrivacyPolicyBanner />, {
+    banner: { showCookiePolicy: false },
+  });
+  const closeButton = getByTestId("closeButton");
+  expect(closeButton).toBeInTheDocument();
+});
```

Comment:
You should add this test  in `should render PrivacyPolicyBanner  when showCookiePolicy is true`, because it should test if the component is rendered with all the right elements